include/votca/xtp/davidsonsolver.h

```diff
@@ -114,7 +107,12 @@ class DavidsonSolver {
         storeNotConvergedData(rep, proj.root_converged, neigen);
         break;
       }
-      extendProjection(rep, proj);
+      Index extension_size = extendProjection(rep, proj);
```

Comment:
Is this the `Index` of the last column before the extension?

Reply:
it tells you by how much the searchspace has been extended.

Reply:
Do you think it needs a better name?

Reply:
that is OK. I forgot how the algorithm works